packages/emitter-framework/src/typescript/components/class-method.tsx

```diff
@@ -17,7 +17,7 @@ export function ClassMethod(props: ClassMethodProps) {
     return <ts.ClassMethod {...props} />;
   }
 
-  const refkey = props.refkey ?? getRefkey(props.type, "method");
+  const refkey = props.refkey ?? efRefkey(props.type, "method");
```

Comment:
I'm not sure we need this ef refkey (do we ever need to reference a class method)? But if we do it should be used in addition to the provided refkey rather than as a default?